services/catalog/src/simcore_service_catalog/core/events.py

```diff
@@ -43,20 +44,14 @@ async def _main_lifespan(app: FastAPI) -> AsyncIterator[State]:
     settings: ApplicationSettings = app.state.settings
 
     yield {
-        PostgresLifespanState.POSTGRES_SETTINGS: settings.CATALOG_POSTGRES,
-        "prometheus_instrumentation_enabled": settings.CATALOG_PROMETHEUS_INSTRUMENTATION_ENABLED,
+        **get_postgres_database_main_lifespan(settings.CATALOG_POSTGRES),
```

Comment:
thx. this is much better. 
I would just rename the function, as I mentioned above, since this does NOT return a lifespan.

Reply:
I would ensure whatever the name of this function becomes, that we use the same everywhere. it makes it easier.